### ISSUE TITLE
Some controls don't visually indicate keyboard focus

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5460]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.461
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- activate controls via Tab key (or arrow key in some scenarios)

#### HelpLink and HelpButton classes have no focus indicators
![2019-09-24_15-46-41](https://user-images.githubusercontent.com/10569626/65556416-75191a80-dee4-11e9-8605-83071a45dfd4.png)

#### SectionChooser class has no focus indicator
The control shows which item is selected, but doesn't have separate indication of keyboard focus. Focus indicator could go on the currently selected item, or around the entire control.
![2019-09-26_10-57-15](https://user-images.githubusercontent.com/10569626/65712691-7402fd00-e04c-11e9-97f8-3d74b91cc361.png)

### Describe the behavior you expected
- expect a visual indication of focus
- such indicators should also meet accessibility contrast requirements, but not calling those out in this issue
